### DISCUSSION TITLE
求 开拓者:拥王者

### DISCUSSION THREAD

**Discussion by github342655596** [Gamuxorg/bbs#244] [投递和请求游戏]
游戏名称：开拓者:拥王者
游戏名称(英文)：Pathfinder: Kingmaker
下载地址 :（如果有的话）：没有

**Comment by shouhuanxiaoji** [comment]
这个游戏有点大啊。。30多G

**Comment by hyh19962008** [comment]
我在上传了

Best

________________________________

**Comment by hyh19962008** [comment]
已上传，另外steam这阵子打折可以考虑一下
